### ISSUE TITLE
[Code reviews] Nits to fix and refactoring

### ISSUE THREAD

**Issue by tompaana** [microsoft/code-with-engineering-playbook#784]
The code reviews section requires some copy-editing and other minor fixes. The items collected here are based on the snapshot of commit [b1497e6](https://github.com/microsoft/code-with-engineering-playbook/commit/b1497e63190344bfb8b6519866b356140ac1a86c). This is not exhaustive list.

Blocking PR: #727 

Generally:

- Improve language in general
- Slashes have whitespaces surrounding them and slashes in general can be removed by rephrasing

FAQ

- Code reviews vs. PRs: Add a covering note before a table
- FAQ titles inconsistent with question marks
- “completed on their own” – rephrase
- “…complex PR on GitHub” – not GitHub specific, remove
- Explain “mob” when mentioned the first time
  - Join the three last (pair/mob) questions if sensible

Evidence and measures

- Folder not required after #782

**Comment by shiranr** [comment]
@tompaana I am not sure that's what you meant in this issue and I might be suggesting something which is too large or doesn't fit. LMK what you think:
I think it is missing an 'intro' instead of the 'code reviews' section, maybe start with the 5 whs:
Why do we need code reviews?
Who will do a code review?
What will be reviewed?
When will it be reviewed?
Where it will be reviewed?? (not sure we need this one 😛 )

and in the different sections we can dive into the topics more. 
For example the 'Inclusion in code review' can be the 'who' section. The  'Pull requests' section can be the 'what'.

A new addition suggestion:
I think each person have different things they look at when they do review. I would actually add a section of different people saying what they look at in a code review, where do they start and main points.

**Comment by tompaana** [comment]
@shiranr I did write this issue (and others) in a hurry and it shows, sorry about that. This issue was meant to collect all small things that aren't right: bad grammar, typos, bad links etc. Not a good practice to have issues like this, but I think we needed this anyway.

You raise excellent points! A good "intro" is indeed something we want, and we have another issue for it: #781 (Another piece of my slobby, hastily done work.) I also think it is important to provide good motivation for why we do code reviews - which I also think should be backed up with data (see #783).

I'm not against restructuring the sections at all. Let's just make a good plan before we execute.

As to your new suggestion: Sounds like D&I focused theme that goes well along with the CSE culture. I'm terribly bad at writing content around that topic. You could work together with @madossan01 who has created couple of related issue for code reviews section.

**Comment by shiranr** [comment]
Thanks @tompaana for clarifying. Sounds good. 
I will take a look again and see if I can find small issues and add them to this ticket.

If there's anything I can take once we are done with this one, please LMK. :)

**Comment by shiranr** [comment]
In addition to what you said:
long sentences should be shortened for example:


in the PR section, fix alignment of General process -> section 2
